main.js

```diff
@@ -2,8 +2,18 @@ const jsonpCallbackNames = [];
 
 const generateCallbackName = () => {
 	const base = 'jsonpCallback';
-	const callbackName = `${base}_${jsonpCallbackNames.length + 1}`;
-	jsonpCallbackNames.push(callbackName);
+	let callbackName;
+	let isUniqueName = false;
+	while (!isUniqueName) {
+		callbackName = `${base}_${jsonpCallbackNames.length + 1}`;
+		isUniqueName = !window.FT[callbackName];
+		if (!isUniqueName) {
+			jsonpCallbackNames.push(null);
```

Comment:
In hindsight, using timestamp + random string as callback name woudl be simpler and resilient enough